odoo/modules/registry.py

```diff
@@ -892,7 +892,11 @@ def signal_changes(self):
             _logger.info("Registry changed, signaling through the database")
             with closing(self.cursor()) as cr:
                 cr.execute("select nextval('base_registry_signaling')")
-                self.registry_sequence = cr.fetchone()[0]
+                # If another process concurrently updates the registry,
+                # the new registry sequence is actually still out-of-date,
+                # and the next call to check_signaling() will detect that and trigger a registry reload.
+                # otherwise, the new registry sequence should be equal to cr.fetchone()[0]
```

Comment:
```suggestion
                # self.registry_sequence will actually be out-of-date,
                # and the next call to check_signaling() will detect that and trigger a registry reload.
                # otherwise, self.registry_sequence should be equal to cr.fetchone()[0]
```